docs/tutorial/tutorial_utils.py

```diff
@@ -13,12 +13,12 @@ class ChatappState(rx.State):
     # Keep track of the chat history as a list of (question, answer) tuples.
     chat_history: list[tuple[str, str]]
 
-    def answer(self):
+    def answer(self) -> None:
```

Comment:
Is it possible to turn off -> None ? I don't think this is a requirement in black